application/controllers/survey_response.py

```diff
@@ -2,13 +2,17 @@
 import time
 import uuid
 
-from application.controllers.helper import is_valid_file_extension, is_valid_file_name_length, \
-    convert_file_object_to_string_base64
-from application.controllers.json_encrypter import Encrypter
-from application.controllers.rabbitmq_submitter import RabbitMQSubmitter
 from flask import current_app
+from sdc.rabbit.exceptions import PublishMessageError
+from sdc.rabbit.publisher import QueuePublisher
 from structlog import get_logger
-from application.controllers.service_helper import get_case_group, get_collection_exercise, get_survey_ref
+
+from application.controllers.helper import (is_valid_file_extension, is_valid_file_name_length,
+                                            convert_file_object_to_string_base64)
+from application.controllers.json_encrypter import Encrypter
+from application.controllers.service_helper import (get_case_group, get_collection_exercise,
```

Comment:
Can stick this on one line and keep flake8 happy would then not need the brackets

Reply:
Do we have an agreed max line-length? I've been using 100 across frontstage/backstage(-api)

Reply:
Do we set the flake8 settings anywhere in the repo? I'm confused how it passes when flake8's default max-line-length is ~80?